.github/workflows/integration_tests.yml

```diff
@@ -356,6 +356,38 @@ jobs:
           name: log-artifact
           path: build-results-desktop-${{ matrix.os }}-${{ matrix.ssl_variant }}*
           retention-days: ${{ env.artifactRetentionDays }}
+      - name: Cleanup Local Copies of Uploaded Artifacts
+        shell: bash
+        run: |
+          rm -rf testapps-desktop-${{ matrix.os }}-${{ matrix.ssl_variant }}
+      - name: Configure GCS Upload Directory (Linux, macOS)
+        if: (startsWith(matrix.os, 'ubuntu') || startsWith(matrix.os, 'macos')) && !cancelled()
+        shell: bash
+        run: echo "GCS_UPLOAD_DIR=${{ github.workspace }}" >> $GITHUB_ENV
+      - name: Configure GCS Upload Directory (Windows)
+        # For Windows move the uploads to C: to create enough space for the
+        # Cloud SDK installation on D:.
+        if: startsWith(matrix.os, 'windows') && !cancelled()
+        shell: bash
+        run: echo "GCS_UPLOAD_DIR=/c/fpl-upload" >> $GITHUB_ENV
+      - name: Move GCS Artifacts to GCS Upload Dir
+        # Copy the artifacts and also the upload scripts since python's relpath
+        # produces errors when called across two different drives. Use tar to
+        # move the files as it neatly deepcopies firestore's symlinks.
+        if:  ${{ (env.GCS_UPLOAD_DIR != github.workspace) && !cancelled() }}
+        shell: bash
+        run: |
+          set -x
+          mkdir -p ${{env.GCS_UPLOAD_DIR}}/scripts/gha/integration_testing
+          mkdir -p ${{env.GCS_UPLOAD_DIR}}/scripts/gha-encrypted
+          tar -chf ${{env.GCS_UPLOAD_DIR}}/ta.tar ta
```

Comment:
Just for readability, could you put a comment here what the "ta" directory is? Others are easy to follow.